index.html

```diff
@@ -234,16 +233,14 @@
               </div>
             </div>
             <!-- third -->
-            <div
-              class="rounded-lg border-for-button py-5 px-4 bg-for-logo-section"
-            >
+            <div class="rounded-lg border-for-button py-5 px-4 bg-[#1A1A1A]">
               <div class="flex flex-col-reverse relative">
                 <div class="grid gap-3 justify-self-end justify-items-center">
                   <img
                     src="./public/assets/homepage/Icon-property.png"
                     alt="Icon-property"
                   />
-                  <p class="font-semibold text-xs text-color text-center">
+                  <p class="font-semibold text-xs text-[#FFFFFF] text-center">
```

Comment:
change it everywhere, I'll not ask it again

Reply:
changed